docs/docs/command-line-interface.mdx

```diff
@@ -321,3 +321,78 @@ The following arguments are available for `rasa x`:
 
 ```text [rasa x --help]
 ```
+
+## Command Line Plugins
+
+You can extend the Rasa Open Source command line interface with custom plugins.
+Plugins extending the `rasa` command line interface can implement new or custom
+behavior which is not included within Rasa Open Source by default.
+
+:::note
+It is currently not possible to overwrite existing `rasa` commands. For example it is
+not possible to use a plugin `rasa-train` to override the `rasa train` command.
+:::
+
+### Installing a Plugin
+
+A command line plugin is nothing more than an executable file, whose name begins with
+`rasa-`. Once it's installed, it is available on the command line via
+`rasa <plugin name>`.
+To install a command line plugin, move its executable somewhere on your `PATH`.
```

Comment:
i'd tell the user either how to view their path, or how to add the directory with the executable to their path

Reply:
maybe by moving the

To make the plugin discoverable, it has to be in your `PATH` environment. You can see
all potential directories in your `PATH` by running `echo $PATH`.
Assuming that `/usr/local/bin` is in your `PATH`, make your plugin discoverable by
moving it to there:

```bash
sudo mv ./rasa-hello  /usr/local/bin
```

Stuff up here and linking from the other one like "install your plugin to make it discoverable". this would make sure users know how to install whether writing their own or using one someone else wrote